.gitlab-ci.yml

```
@@ -0,0 +1,60 @@
+workflow:
+  rules:
+    # For merge requests, create a pipeline.
+    - if: '$CI_MERGE_REQUEST_IID'
+    # For `master` branch, create a pipeline (this includes on schedules, pushes, merges, etc.).
+    # - if: '$CI_COMMIT_BRANCH == $CI_DEFAULT_BRANCH'
+    # Temporarly create Ci for every branch:
+    - if: '$CI_COMMIT_BRANCH'
+    # For tags, create a pipeline.
+    - if: '$CI_COMMIT_TAG'
+    # For stable branches, create a pipeline.
+    - if: '$CI_COMMIT_BRANCH =~ /^[\d-]+-stable$/'
+    #
+
+default:
+  image: golang:1.13
```

We can update the version we test to `1.15` and `1.14`, generally I just test the last 2 versions, but have been slacking on updating the existing CI
```suggestion
  image: golang:1.15
```

---

@zaquestion I will add 1.15 😄 The README still describes 1.13 as the minimal requirement, and I thought it might be best to not change it all in one go (pun intended) 😆

---

Eh, in thinking about this more, I'd only test 1.15 (the version we'll use to build the release), but we can also verify that `1.14` is able to build the project. We can update the readme to set `1.14` as the minimum.